tests/hpc/mpi_master.pm

```diff
@@ -18,23 +18,26 @@ use warnings;
 use testapi;
 use lockapi;
 use utils;
-use registration qw(add_suseconnect_product get_addon_fullname);
```

Comment:
Why do you remove this? You still use `add_suseconnect_product` in line 44 (when version is sle-15 or opensuse)